consensus/scp/tests/mock_network/cyclic_topology.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2018-2020 MobileCoin Inc.
+
+// Ring style network topologies.
+
+// We allow dead code because not all integration tests use all of the common code.
+// https://github.com/rust-lang/rust/issues/46379
+#![allow(dead_code)]
+
+use crate::mock_network;
+
```

Comment:
We intentionally remove all newlines between `use` statements so all imports are sorted by rustfmt.

```suggestion
```